src/event-notion/index.ts

```diff
@@ -132,7 +132,7 @@ export const syncHostFormToNotionCalendar = async (config: EventNotionPipelineCo
   // The Notion API is easy enough to get.
   const notion = await getNotionAPI(config.notionToken);
   // Get the Discord webhook as well.
```

Comment:
Kind of a picky thing but could change comment to say "Get Discord Channel ID..."

Reply:
Good catch, just updated!